src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,29 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+  let catYears = 0;
+  let dogYears = 0;
+  let result = [];
+
+  for (let i = 1; i < number + 1; i++) {
+    if (i === 1) {
+      catYears = 15;
+      dogYears = 15;
+    }
+
+    if (i === 2) {
+      catYears = 15 + 9;
+      dogYears = 15 + 9;
+    }
+
+    if (i >= 3) {
+      catYears += 4;
+      dogYears += 5;
+    }
+  }
+  result[0] = number;
+  result[1] = catYears;
+  result[2] = dogYears;
+
+  return result;
```

Comment:
A more shorthand way of doing this would be to return the array directly, this would also mean you no. longer have to declare the `result` variable at the top of the file:
```

  return [number, catYears, dogYears];
```